src/components/axes/categoryAxis.ts

```diff
@@ -135,13 +129,15 @@ export module Axes {
      * @param {string[]} ticks The strings that will be printed on the ticks.
      */
     private _measureTicks(axisWidth: number, axisHeight: number, scale: Scales.Category, ticks: string[]) {
+      var axisSpace = this._isHorizontal() ? axisWidth : axisHeight;
+      var expectedRangeBand = axisSpace / (2 * scale.outerPadding() + (ticks.length - 1) * scale.innerPadding() + ticks.length);
```

Comment:
It's `expectedRangeBand` because the rangeBand of the scale cannot be calculated until the range is set.  Thus, it is an expected `rangeBand` of the scale "if it had the range given by the `axisSpace variable`.

The math behind it is yeah a bit complexish:  I'll explain below.
 
The entirety of a `Scales.Category` can consist of some number of bands as well as paddings in-between the bands as well as paddings between the edge of the scale and the outer bands.

Thus,
let Wb = bandWidth
let Wo = outerPaddingWidth
let Wi = innerPaddingWidth
let Nb = number of bands
let Wt = total width

Thus, we get the following equation:
Wt = 2 * Wo + (Nb - 1) * Wi + Nb * Wb

Now by definition of `Scales.Category`, the width of each of the paddings is equivalent to the respective padding proportion on the scale multiplied by the bandWidth.

So,
let Pi = innerPadding = Wi / Wb
let Po = outerPadding = Wo / Wb

And thus,
Wt = 2 * Po * Wb + (Nb - 1) * Pi * Wb + Nb * Wb

And since we are solving for Wb, we just need to do a bit of algebra...
Wb = Wt / (2 * Po + (Nb - 1) * Pi + Nb)